olive/auto_optimizer/config_template/available_pass_flows.yaml

```diff
@@ -0,0 +1,79 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+_variables:
+  - &fp32 ["OnnxConversion", "OrtTransformersOptimization", "OrtPerfTuning"]
+  - &fp16 ["OnnxConversion", "OrtTransformersOptimization", "OrtPerfTuning"]
```

Comment:
Why not invert this mapping? List precision supported by each Pass.

Reply:
Oh, the `&fp32`, `&inc_int8` and etc are just the reference to the pass_flows template which also are mapped with accelerate/ep(recall the pass flows table we talked before.)

Then this yaml file is used to describe the table I shared. It is the template which olive will retrieve according to accelerator/ep/precisions.
Revertting this mapping will flatten the pass_flows into pass level. Somehow we have to composite the pass into pass flows if revertting happens.

Reply:
Updated.